x-pack/plugins/maps/public/index_pattern_util.ts

```diff
@@ -98,7 +98,6 @@ export function supportsGeoTileAgg(field?: IndexPatternField): boolean {
 export function getSourceFields(fields: IndexPatternField[]): IndexPatternField[] {
   return fields.filter((field) => {
     // Multi fields are not stored in _source and only exist in index.
-    const isMultiField = field.subType && field.subType.multi;
-    return !isMultiField && !indexPatterns.isNestedField(field);
+    return !field.isSubtypeMulti() && !field.isSubtypeNested();
```

Comment:
Looks like `isSubtypeNested` has been added to field. Does this mean that `indexPatterns.isNestedField` is no longer needed?

Reply:
Preferably you're working with an `IndexPatternField` and can use the class method. There are numerous places in the code base working with the `IFieldType` instead so `indexPatterns.isNestedField` is needed.